native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -19,17 +19,12 @@
 import {
   View,
   TextInput,
-  Image,
   Platform,
   StyleSheet,
 } from 'react-native';
 import React from 'react';
 import PropTypes from 'prop-types';
-
-const tickSmIcon = require('./icons/sm/tick-circle.png'); // eslint-disable-line import/no-unresolved
-const tickLgIcon = require('./icons/lg/tick-circle.png'); // eslint-disable-line import/no-unresolved
-const exclaimationSmIcon = require('./icons/sm/exclaimation-circle.png'); // eslint-disable-line import/no-unresolved
-const exclaimationLgIcon = require('./icons/lg/exclaimation-circle.png'); // eslint-disable-line import/no-unresolved
+import BpkIcon from 'react-native-bpk-component-icon';
```

Comment:
I'm assuming this PR is introducing both the component and the implementation in BpkTextInput, is that correct?

Reply:
Yeah - I implemented its use where appropriate. It does mean I've somewhat bloated the PR, but I wanted to make the changes in order to test it and I thought I might as well include what I'd done

Reply:
Yeah actually we have a separate card for consuming the BpkIcon component in other Bpk components - i'd take the BpkTextInput & BpkButton changes out of this MR - it's too early.

Reply:
@matthewdavidson also made a good point that if we make use of `BpkIcon` before it is finished, we will actually break the existing react-native components as the font won't yet be installed in the apps